site/js/api.js

```diff
@@ -1,5 +1,13 @@
-export function post(path, dataObj) {
-  return fetch('/api/' + path, {
+// These functions assume that serverURL is running
+// on standard HTTP - this will likely change in the
+// future (e.g. Service Workers only work on https).
+//
+// Note that WebSockets use their own protocols:
+//   http  <--> ws
+//   https <--> wss
+
+export function post(path, dataObj, serverURL) {
+  return fetch('http://' + serverURL + '/api/' + path, {
```

Comment:
Could you just fetch `'//' + serverURL`? That way the protocol is automagically picked based on whatever protocol you're using to connect to the client. I guess that wouldn't make `ws://`/`wss://` be automatically picked, though.

Reply:
We should enforce HTTPS soon anyway - it's needed for #6.